ibverbs-sys/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "ibverbs-sys"
+version = "0.4.2"
+description = "Direct bindgen bindings for RDMA ibverbs through rdma-core used by the ibverbs crate"
+authors = [
+    "Jon Gjengset <jon@thesquareplanet.com>",
+    "Jonathan Behrens <fintelia@gmail.com>"
+]
+edition = "2018"
+build = "build.rs"
+links = "ibverbs"
+
+keywords = ["rdma", "ibverbs", "networking", "ffi"]
+categories = ["network-programming", "api-bindings"]
+
+license = "MIT/Apache-2.0"
+
+[build-dependencies]
+bindgen = "0.57"
```

Comment:
Let's do `0.58` while we're at it ;)

Reply:
Good lord has it been that long since I started it?